OpenTAP.TUI/PropEditProviders/FilePathEditProvider.cs

```diff
@@ -26,10 +27,13 @@ public View Edit(AnnotationCollection annotation)
                 try
                 {
                     var value = annotation.Get<IObjectValueAnnotation>().Value;
```

Comment:
I think we need to check if cancel was called otherwise dialog.FilePath is null.

```suggestion
                    if (dialog.Canceled) return;
                    var value = annotation.Get<IObjectValueAnnotation>().Value;
```